distributed3/core.py

```diff
@@ -84,32 +93,29 @@ def connect_sync(host, port, timeout=1):
     return s
 
 
-def write_sync(s, msg):
+def write_sync(sock, msg):
     if not isinstance(msg, bytes):
         msg = dumps(msg)
-    s.send(struct.pack('L', len(msg)))
-    s.send(msg)
+    sock.send(msg + sentinel)
```

Comment:
does it make sense to make two send calls, instead of performing bytestring concatenation here?